code/datums/status_effects/debuffs.dm

```diff
@@ -9,6 +9,39 @@
 	if(owner.reagents)
 		owner.reagents.del_reagent("holywater") //can't be deconverted
 
+/datum/status_effect/crusher_mark
+	id = "crusher_mark"
+	duration = 300 //if you leave for 30 seconds you lose the mark, deal with it
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	var/mutable_appearance/marked_underlay
+	var/obj/item/twohanded/kinetic_crusher/hammer_synced
+
```

Comment:
I'll raise the same concern here that I did with CornMyCob's PR: I know the crusher-mark status effect for carbon mobs is commented out in current code: would this code let people use the crusher mark against carbon mobs?

Reply:
You can apply it to anything that's living.

I don't see how this is a problem, personally.

Reply:
I am pretty sure that you cant use the crusher's mark in pressurised environment so miners cant use this on station most of the time, I think its fine.

Reply:
Does mean that ash walkers that get a crusher somehow become much, much more dangerous

Reply:
It does. :eyes:

Reply:
It could also mean that you could use the crusher on necropolis tendrils, maybe, making a quick rushdown of goliath tendrils much more feasible.